README.md

```diff
@@ -25,15 +26,30 @@ The files committed in the `shared_files/encrypted` directory hold secret inform
 - Create a file named `.vault_password` in the root directory of the repository
 - Place the Ansible Vault password in this file
 
-#### Git hook
 
-To avoid committing and pushing unencrypted secret files, use the included `scripts/pre-commit` git pre-commit hook by running the following script from the repository root directory:
+#### Installing and Using Pre-commit
 
+Anyone committing to this repo must use the pre-commit hook to lower the likelihood that secrets will be exposed.
+
+##### Step 1: Install pre-commit
+
+You can install pre-commit using the MacOS package manager Homebrew:
+
+```sh
+brew install pre-commit
 ```
-cp ops/pre-commit .git/hooks
+
+Other installation options can be found in the [pre-commit documentation](https://pre-commit.com/#install).
+
+##### Step 2: Install the hooks
+
+Run the following command to install the gitleaks hook:
+
+```sh
+pre-commit install
 ```
 
-The pre-commit hook will also ensure that any added, copied, or modified go files are formatted properly.
+This will download and install the pre-commit hooks specified in `.pre-commit-config.yaml`.
```

Comment:
Maybe keep some detail from the deleted line like

```suggestion
This will download and install the pre-commit hooks specified in `.pre-commit-config.yaml`, which includes gitleaks for secret scanning and go-imports to ensure that any added, copied, or modified go files are formatted properly.
```